.mergify.yml

```diff
@@ -541,4 +472,4 @@ pull_request_rules:
         add:
           - Repo activity
           - ci/skip/e2e
-          - ci/skip/multi-arch-build
+          - ci/skip/multi-arch-build
```

Comment:
it looks like a new line is missing here, can you please check

Reply:
done